### PR TITLE
Set the resources cpu and memory for global hub components

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>
Ref: https://issues.redhat.com/browse/ACM-6381

Note: https://home.robusta.dev/blog/stop-using-cpu-limits

### CODE REVIEW

File: operator/pkg/controllers/hubofhubs/manifests/manager/deployment.yaml [hunk 1]
```diff
@@ -19,6 +19,12 @@ spec:
       containers:
         - name: multicluster-global-hub-manager
           image: {{.Image}}
+          resources:
+            requests:
+              memory: 100Mi
+              cpu: 100m
```
Comment (clyang82) [inline operator/pkg/controllers/hubofhubs/manifests/manager/deployment.yaml:25]: 100m? is it correct based on your tests?

Comment (yanmxa) [reply operator/pkg/controllers/hubofhubs/manifests/manager/deployment.yaml:25]: image
Yes! This is the memory consumption trend.

File: operator/config/manager/manager.yaml [hunk 2]
```diff
@@ -75,10 +75,11 @@ spec:
         # More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
         resources:
           limits:
-            cpu: 500m
-            memory: 512Mi
+            # cpu: 50m
+            # https://home.robusta.dev/blog/stop-using-cpu-limits
+            memory: 200Mi
```
Comment (ldpliu) [inline operator/config/manager/manager.yaml:80]: I think the memory limit is too small, it's better to set it to 1Gi/2Gi
@clyang82 What's your opinion?

File: operator/pkg/controllers/hubofhubs/manifests/manager/deployment.yaml [hunk 3]
```diff
@@ -19,6 +19,12 @@ spec:
...
+            limits:
+              memory: 300Mi
```
Comment (ldpliu) [inline operator/pkg/controllers/hubofhubs/manifests/manager/deployment.yaml:27]: I think the memory limit is too small, it's better to set it to 1Gi/2Gi
@clyang82 What's your opinion?

**Comment by yanmxa** [comment]
@ldpliu The values of manager and operator memory consumption are based on test results. ref: https://github.com/yanmxa/hub-of-hubs/tree/br_scale_test/doc/simulation

**Pull request merged**